inject.js

```diff
@@ -56,8 +72,20 @@ const updateElem = async () => {
     || (options.disabledOnPages.watch && window.location.pathname === '/watch')
     || (options.disabledOnPages.subscriptions && window.location.pathname === '/feed/subscriptions');
 
-  elem.innerHTML = `/* Injected by the Hide YouTube Thumbnails extension */
-  ${css[isDisabled ? 'normal' : options.thumbnailMode]}`
+  if(options.workMode.enabled){
```

Comment:
must: I think we can move this logic into the `isDisabled` check. For example, adding a condition like:

```
    || (options.workMode.enabled && !IsItWorkTime(options.workMode.startTime, options.workMode.endTime))
```

This significantly reduces code duplication and complexity.